### PR TITLE
Copy source schemas and add a readme

### PR DESCRIPTION
**Pull request by sgillies**
I'm inclined to keep the various JSON schemas related to PXM in one place, yes @normanb ?

### CODE REVIEW

File: schemas/pxm-manifest-0.5.0.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://mapbox.com/schemas/pxm-manifest-0.5.0.json",
```
Comment (sgillies) [inline schemas/pxm-manifest-0.5.0.json]: According to https://spacetelescope.github.io/understanding-json-schema/basics.html#declaring-a-json-schema, the `$schema` and `id` items are standard practice.

These URLs don't exist yet, of course.

**Comment by sgillies** [comment]
Ready for review. Note that this is based on the json-schema branch.

File: schemas/pxm-manifest-0.5.0.json (hunk 1 above)
Comment (vincentsarago) [reply schemas/pxm-manifest-0.5.0.json]: @sgillies can you point to the GitHub url instead to `mapbox.com` ?

File: schemas/pxm-manifest-0.5.0.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "id": "https://mapbox.com/schemas/pxm-manifest-0.5.0.json",
+  "id": "https://raw.githubusercontent.com/mapbox/pxm-manifest-specification/schemas/pxm-manifest-0.5.0.json",
```
Comment (sgillies) [inline schemas/pxm-manifest-0.5.0.json]: Comme ça @vincentsarago ?

Comment (vincentsarago) [reply schemas/pxm-manifest-0.5.0.json]: c'est parfait 👍

**Pull request merged**